app/models/like.rb

```diff
@@ -0,0 +1,10 @@
+class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post
+
+  after_create :update_likes_count
+
+  def update_likes_count
+    post.update(likes_counter: post.likes.count)
+  end
```

Comment:
- [ ] if you are using this here, so you need to remove the same method you put in your post model

Reply:
@VICTOR-CHUKWU  Hi! I don't get why, because on the diagram the comments and likes updates should go on both tables, for each case

Reply:
hello kindly check your project requirement
```
Your models should include some additional methods.

Users
A method that returns the 3 most recent posts for a given user.
Posts
A method that updates the posts counter for a user.
A method which returns the 5 most recent comments for a given post.
Comments
A method that updates the comments counter for a post.
Likes
A method that updates the likes counter for a post.
```